src/template/args.json

```diff
@@ -1,33 +1,21 @@
 {
-    "$schema": "https://repometric.github.io/linterhub/schema/args.json",
-    "name": "",
-    "type": "object",
+    "$schema": "https://schema.linterhub.com/args.json",
+    "$id": "",
+    "$version": "1.0.0",
+    "package": "",
+    "description": "",
+    "delimiter": "=",
     "allOf": [
         {
-            "$ref": "#/definitions/arguments"
+            "$ref": "./schemaver.json"
```

Comment:
As far as there is no reason to copy schemaver to this repo you may use an absolute uri to the latest version.